scm-ui/ui-webapp/src/containers/Profile.tsx

```diff
@@ -72,9 +72,18 @@ class Profile extends React.Component<Props> {
     return !!me?._links?.apiKeys;
```

Comment:
```suggestion
    canManageApiKeys = () => !!this.props.me?._links?.apiKeys;
```
Maybe shorten this code?